dbstore.go

```diff
@@ -97,11 +134,38 @@ type rSliceElement struct {
 
 var tableToEntities = map[string]interface{}{
 	"worksheets":               &rWorksheet{},
+	"worksheet_edits":          &rEdit{},
 	"worksheet_values":         &rValue{},
 	"worksheet_parents":        &rParent{},
 	"worksheet_slice_elements": &rSliceElement{},
 }
 
+func (s *Session) Edit(editId string) (time.Time, map[string]int, error) {
+	var editRecs []rEdit
+	if err := s.tx.
+		Select("*").
+		From("worksheet_edits").
+		Where("edit_id = $1", editId).
+		OrderBy("worksheet_id, to_version").
```

Comment:
we talked about this. the orderby made me think we were dealing with multiple records for a given worksheet, but now i see that with edit id we should only see a given worksheet one time, by constraint.

i was mostly concerned about the touchedWs map, but given the constraint that map should not see the same key twice written (perhaps assert?).